src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -12,19 +12,19 @@
             cursor: pointer;
         }
 
-        .signpost-action {
-            min-width: $clr_baselineRem_1;
-            margin: 0;
-            padding: 0;
+        .signpost-trigger {
             color: $clr-dark-midtone-gray;
-            outline: none;
 
-            clr-icon {
-                width: $clr_baselineRem_1;
-                height: $clr_baselineRem_1;
+            &.btn-icon {
+                border: none;
```

Comment:
I am a bit confused by this. So if the trigger is a Clarity outline button, there is no border for it?

Reply:
I'm using a btn.btn-icon which has a border. The specs do not have a border around any of the trigger examples.